README.md

```diff
@@ -10,4 +10,4 @@ Autover provides:
      location for each Python package specified; see the [autover
      script](https://github.com/ioam/autover/blob/master/scripts/autover).
 
-Authors: Jean-Luc Stevens, Chris Ball and James A. Bednar
+Authors: Jean-Luc Stevens and James A. Bednar
```

Comment:
Sneaky!

Reply:
I'm sure Jim didn't appreciate having his name so close to a missing comma.

Reply:
Ah, that is a very good point!